packages/pouchdb-adapter-node-websql/src/index.js

```diff
@@ -7,7 +7,7 @@ function NodeWebSqlPouch(opts, callback) {
     websql: websql // pass node-websql in as our "openDatabase" function
   }, opts);
 
-  WebSqlPouch.call(this, _opts, callback);
+  WebSqlPouchCore.call(this, _opts, callback);
```

Comment:
When this wasn't calling `core`, it was just uselessly calling what `pouchdb-adapter-websql` exports, which is the `PouchDB.plugin()` function.